packages/vanchor-client/src/example-queries.ts

```diff
@@ -12,16 +12,30 @@ import {
   GetVAnchorTotalValueLockedByChainAndByTokenDayInterval,
   GetVAnchorsTotalValueLockedByChainDayInterval,
   GetVAnchorsTotalValueLockedByChainsDayInterval,
+  GetVAnchorsTVLByChainByDateRange,
 } from './queries/tvl';
+import {
+  GetVAnchorDepositByChainDayInterval,
+  GetVAnchorDepositByChainAndByTokenDayInterval,
+  GetVAnchorsDepositByChainDayInterval,
+  GetVAnchorsDepositByChainsDayInterval,
+  GetVAnchorsDepositByChainByDateRange,
+} from './queries/deposit';
 import { DateUtil } from './utils/date';
 
+const vAnchorAddress = '0x91eb86019fd8d7c5a9e31143d422850a13f670a3';
+const subgraphUrl = SubgraphUrl.vAnchorAthenaLocal;
+
 async function main() {
-  // Get total value locked by a vAnchor on a chain
-  await runTvlQueries();
-  // Total Value Locked Queries of 15 mins interval
-  await runTvl15MinsIntervalQueries();
-  // Day Interval
-  await runTvlDayIntervalQueries();
+  // // Get total value locked by a vAnchor on a chain
```

Comment:
Please don't leave commented out code in repo, this is sloppy @vutuanlinh2k2 @AtelyPham.